tests/dummy/app/templates/public-pages.hbs

```diff
@@ -7,19 +7,25 @@
       <a href="https://github.com/cibernox/ember-power-calendar" class="main-header-nav-link">Github</a>
     </div>
     <div class="main-header-logo">
-      <a href={{href-to 'public-pages.index'}} class="home-link">
+      <LinkTo
+        @route="public-pages.index"
```

Comment:
Not needed, not against either.

Reply:
Yeah... Sorry, sometimes I go a bit overboard.